modules/ROOT/edits/install-self-managed.adoc

```diff
@@ -0,0 +1,259 @@
+
+[%header,cols="2*a"]
+.Cerfificate configuration for Self-Managed Kubernetes
+|===
+| Control Plane | Certificates
+| US control plane | transport-layer.prod.cloudhub.io +
+configuration-resolver.prod.cloudhub.io
+| EU control plane | transport-layer.prod-eu.msap.io +
+configuration-resolver.prod-eu.msap.io
+|===
+
+== Step 2: Create a Runtime Fabric using Runtime Manager
+
+[IMPORTANT]
+====
+The procedures in this section should be performed by a MuleSoft organization administrator.
+====
+
+To install Runtime Fabric on Self-Managed Kubernetes, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
++
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Accept*.
++
+Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section.
+
+
+== Step 3: Download the rtfctl Utility
+
+[IMPORTANT]
+====
+The tasks in the section must be performed by an IT administrator.
+====
+
+Runtime Fabric on Self-Managed Kubernetes uses the `rtfctl` command-line utility for installation and management tasks. See xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool].
+
+
+. Download the `rtfctl` command-line utility:
++
+`rtfctl` is supported on Windows, MacOS (Darwin), and Linux. Download this utility using the URLs below:
++
+*Windows:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-windows/latest -o rtfctl.exe
+----
++
+*MacOS (Darwin):*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-darwin/latest -o rtfctl
+----
++
+*Linux:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
+
+. Change file permissions for the `rtfctl` command-line utility:
++
+----
+sudo chmod +x rtfctl
+----
+
+== Step 4: Install Runtime Fabric
+
+[IMPORTANT]
+====
+The procedures in the section must be performed by an IT administrator.
+====
+
+After creating a Runtime Fabric in Runtime Manager and obtaining the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command-line utility.
+
+If your Kubernetes configuration is not located in the `\~/.kube/config` directory, set the `KUBECONFIG` environment variable before running `rtfctl`:
+----
+export KUBECONFIG=<path-to-kubeconfig>
+----
+
+
+. Validate that your Kubernetes environment is read for installation:
```

Comment:
"Ensure"?
Even though the syntax uses the word "validate," to me, validation is something that you do AFTER your change is made, not before it to enable it....